panacea/query_client.go

```diff
@@ -112,6 +123,30 @@ func NewQueryClient(ctx context.Context, config *config.Config, info TrustedBloc
 	}, nil
 }
 
+// refresh update light block, when the last light block has been updated more than trustPeriod * 2/3.
+func refresh(ctx context.Context, lc *light.Client, trustPeriod time.Duration) error {
+	logrus.Info("check latest light block")
+	lastBlockHeight, err := lc.LastTrustedHeight()
+	if err != nil {
+		return err
+	}
+	lastBlock, err := lc.TrustedLightBlock(lastBlockHeight)
+	if err != nil {
+		return err
+	}
+	lastBlockTime := lastBlock.Time
+	currentTime := time.Now()
+	timeDiff := currentTime.Sub(lastBlockTime)
+	if timeDiff > trustPeriod*2/3 {
+		logrus.Info("update latest light block")
+		_, err = lc.Update(ctx, time.Now())
+		if err != nil {
+			return err
+		}
```

Comment:
It would be good to change it like this.
```suggestion
		if _, err := lc.Update(ctx, time.Now()); err != nil {
			return err
		}
```

Reply:
Thanks, I fixed it in [2235390](https://github.com/medibloc/panacea-doracle/pull/21/commits/22353900e45ca233fe82c98c1cc666f27fc3b703).